src/ray/raylet/scheduling/local_resource_manager.cc

```diff
@@ -342,6 +342,27 @@ std::optional<syncer::RaySyncMessage> LocalResourceManager::CreateSyncMessage(
 
   resources_data.set_is_draining(IsLocalNodeDraining());
 
+  std::stringstream node_activity;
+  for (const auto &iter : last_idle_times_) {
+    if (iter.second == absl::nullopt) {
+      // If it is a WorkFootprint
+      if (iter.first.index() == 0) {
+        switch (std::get<WorkFootprint>(iter.first)) {
+        case WorkFootprint::NODE_WORKERS:
+          node_activity << " Node currently has leased workers." << std::endl;
+          break;
+        default:
+          UNREACHABLE;
+        }
+        // If it is a ResourceID
+      } else {
+        node_activity << " Resource: " << std::get<ResourceID>(iter.first).Binary()
```

Comment:
will this show up as non-readable string?

Reply:
Could we add the test on this in the local_resource_manager_test where we are already testing the idle time/non idle time?